R/dash.R

```diff
@@ -6,7 +6,10 @@
 #' app <- Dash$new(
 #'   name = "dash",
 #'   server = fiery::Fire$new(),
-#'   static_folder = NULL,
+#'   static_folder = 'static',
```

Comment:
Can we get rid of `static_folder` now, so we don't create a breaking change down the line like we're going to have for dashpy? https://github.com/plotly/dash/issues/469#issuecomment-450355079

Reply:
Yes, let’s :hocho: this.

Reply:
fixed in 16c4efa9d9c08a0b07c3b6483bb0ba220f589303